checkbox-provider-kivu/bin/compare_power_consumption.py

```diff
@@ -34,27 +45,27 @@ def load_energy(fname: str ) -> tuple[float, dict[str, float]]:
     no_hwacc_energy_file = sys.argv[2]
     min_gain = float(sys.argv[3])
 
+    # data format
+    # Example: (600.097183, {'psys': 9976.218114, 'package-0': 3658.829745, 'core': 825.515099, 'uncore': 318.009806})
     hwacc_energy = load_energy(hwacc_energy_file)
     no_hwacc_energy = load_energy(no_hwacc_energy_file)
-    
-    print(f'Enabled: {hwacc_energy}')
-    print(f'Disabled: {no_hwacc_energy}')
 
-    gain_energy = {}    
-    for key, value in hwacc_energy[1].items():
-        hwacc_joule_per_sec = value / hwacc_energy[0]
-        no_hwacc_joule_per_sec = no_hwacc_energy[1][key] / no_hwacc_energy[0]
-        if no_hwacc_joule_per_sec != 0:
-            gain = ((no_hwacc_joule_per_sec - hwacc_joule_per_sec) / no_hwacc_joule_per_sec) * 100
-        else:
-            gain = -1000000 * hwacc_joule_per_sec
-        gain_energy[key] = gain
+    hwacc_cons = get_energy_usage(hwacc_energy[1])
+    no_hwacc_cons = get_energy_usage(no_hwacc_energy[1])
+
+    print(f'Enabled: {hwacc_energy} - Consumption : {hwacc_cons}')
+    print(f'Disabled: {no_hwacc_energy} - Consumption : {no_hwacc_cons}')
 
-    avg_gain = statistics.mean(gain_energy.values())
+    hwacc_joule_per_sec = hwacc_cons / hwacc_energy[0]
+    no_hwacc_joule_per_sec = no_hwacc_cons / no_hwacc_energy[0]
+    if no_hwacc_joule_per_sec != 0:
+        gain = ((no_hwacc_joule_per_sec - hwacc_joule_per_sec) / no_hwacc_joule_per_sec) * 100
+    else:
+        gain = -1000000 * hwacc_joule_per_sec
```

Comment:
s/1000000/1_000_000

You're already using f-strings, so 3.6 python is required, meaning that you can use number literal separator.